packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -683,7 +683,8 @@
 </section>
 
 <section>
-  <h3 class="dummy-h3">Design guidelines</h3>
+  <h3 class="dummy-h3" id="design-guidelines"><a href="#design-guidelines" class="dummy-link-section">§</a>
```

Comment:
adding a comment here: if you check, I think one has been missed in this file ("accessibility" section)